content/en/tracing/_index.md

```diff
@@ -12,41 +21,46 @@ aliases:
-{{< /whatsnext >}}
+ Tracing allows you to see a single request moving across your system and infrastructure, and it gives you systematic data about precisely what is happening to this request. This gives you better insight into your systems interdependencies, it allows you to see where requests are taking the longest, it gives you visual cues for accurately troubleshooting outages and other system issues, and it allows you to see detailed traces of individual requests-side by side with your logs and infrastructure monitoring.
 
-## Terminology
+## Setting up APM
 
-| Term          | Definition                                                          | Note                                                                                                                                             |
-| :----         | :-----                                                              | :---                                                                                                                                             |
-| [Service][1]  | Name of a set of processes that do the same job                     | Services are displayed on the [Datadog Services list][2] and have [out of the box performances graphs][3].                                       |
-| [Resource][4] | Particular action for a service                                     | Resources are available on the [Resources list for each service][5] and have [out of the box performances graphs][6]                             |
-| [Trace][7]    | Representation of a request as it flows across a distributed system | A trace can be collected in [any language][8]. Traces are found in the [Traces list for each resources][9] or in the [Trace search directly][10] |
-| [Span][11]    | A logical unit of work in the system                                | Spans are associated with a [service][1] and optionally a [resource][4]. Each span consists of a start time, a duration, and optional tags.      |
+The application code instrumentation sends traces to the Agent every second. The Agent then calculates performances metrics over all traces and send those metrics along with [a sample of traces][2] to Datadog every 10 seconds. To get APM up, running, and reporting data from your application, follow these steps:
 
+1. **[Send traces to Datadog][3]**:
+   Install and configure the latest [Datadog Agent][4]. (On macOS, install and run the Trace Agent in addition to the Datadog Agent. See the [macOS Trace Agent][5] documentation for more information). APM is enabled by default in Agent 6, however there are additional configurations to be set in a containerized environment including setting `apm_non_local_traffic: true`. To get an overview of all the possible settings for APM including setting up APM in containerized environments such as [Docker][6] or [Kubernetes][7], get started [Sending traces to Datadog][3].
 
+2. **[Configure your application to send traces to your Datadog Agent][8]**:
```

Comment:
This is the send traces to Datadog Agent step

Reply:
this might be better as #1 i think

Reply:
see my response to your comment in the mainnav